src/main/java/com/entropyteam/entropay/employees/dtos/EmployeeDto.java

```diff
@@ -15,46 +15,39 @@
-                          String country,
-                          @NotNull(message = "Personal Number is mandatory")
-                          String personalNumber,
-                          String taxId,
-                          String emergencyContactFullName,
-                          String emergencyContactPhone,
-                          List<UUID> profile,
-                          String notes,
-                          String healthInsurance,
-                          List<PaymentInformationDto> paymentInformation,
-                          List<UUID> technologies,
-                          @Email
-                          String labourEmail,
-                          @JsonFormat(pattern = "yyyy-MM-dd") LocalDate birthDate,
-                          @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss") LocalDateTime createdAt,
-                          @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss") LocalDateTime modifiedAt,
-                          boolean deleted,
-                          String project,
-                          String client,
-                          String role,
-                          UUID lastAssignmentId,
-                          @JsonFormat(pattern = "yyyy-MM-dd") LocalDate startDate,
-                          @NotNull(message = "Active state is mandatory")
-                          boolean active,
-                          Integer availableDays) {
+        @NotNull(message = "Internal ID is mandatory") String internalId,
+        @NotNull(message = "First Name is mandatory") String firstName,
+        @NotNull(message = "Last Name is mandatory") String lastName,
+        @Email @NotNull(message = "Email is mandatory") String personalEmail,
+        String phoneNumber,
+        String mobileNumber,
+        String address,
+        String city,
+        String state,
+        String zip,
+        String country,
+        @NotNull(message = "Personal Number is mandatory") String personalNumber,
+        String taxId,
+        String emergencyContactFullName,
+        String emergencyContactPhone,
+        List<UUID> profile,
+        String notes,
+        String healthInsurance,
+        List<PaymentInformationDto> paymentInformation,
+        List<UUID> technologies,
+        @Email String labourEmail,
+        @JsonFormat(pattern = "yyyy-MM-dd") LocalDate birthDate,
+        @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss") LocalDateTime createdAt,
+        @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss") LocalDateTime modifiedAt,
+        boolean deleted,
+        String project,
+        String client,
+        String role,
+        UUID lastAssignmentId,
+        @JsonFormat(pattern = "yyyy-MM-dd") LocalDate startDate,
+        @JsonFormat(pattern = "yyyy-MM-dd") LocalDate endDate,
+        @NotNull(message = "Active state is mandatory") boolean active,
+        Integer availableDays) {
 
     public EmployeeDto(Employee employee, List<PaymentInformation> paymentInformationList) {
```

Comment:
Are we using this method?

Reply:
Only this one is being used "@NotNull(message = "Active state is mandatory") boolean active, Integer availableDays)" the rest appear unused